### DISCUSSION TITLE
File picker on chromium won't work on niri.

### DISCUSSION THREAD

**Discussion by lidgnulinux** [YaLTeR/niri#257] [Q&A] [answered]
Hello, I just need a confirmation. Is it just me or the file picker on chromium browser doesn't work when we use niri ? Other apps's file picker seems work fine. Strangely enough, when I run chromium from command line (with exact same command from desktop file), the file picker works. 

Niri : 0.1.3 (acd3365)
Chromium : 122.0.6261.57

**Comment by YaLTeR** [comment]
That's odd, works for me (with Flatpak Chromium).

**Reply by lidgnulinux** [reply]
I'm using chromium from alien repo (I'm using slackware).

**Comment by FreeFull** [comment]
Do you have `xdg-desktop-portal-gtk` installed? I'm assuming chromium might be trying to use xdg-desktop-portal for the file picker

**Reply by lidgnulinux** [reply]
In my opinion, we don't need to focus on that, without xdg-desktop-portal-gtk file picker works. The problem is when I launch using wofi (launcher), file picker won't work, but if I launch using command line, it works.

**Comment by lidgnulinux** [comment, marked as answer]
I end up make a shortcut / keybind rather than using launcher. Closing it now. Thanks.
